recipes/zulu-openjdk/all/conandata.yml

```diff
@@ -1,14 +1,36 @@
 sources:
   "11.0.8":
```

Comment:
Let's also add arch to the 11.0.8 "sources".
Then there's no need for change in logic between versions.

Reply:
M1 does not for the old version

Reply:
Indeed. Then only add x86_64,